dask/array/core.py

```diff
@@ -140,6 +142,34 @@ def getter_inline(a, b, asarray=True, lock=None):
     return getter(a, b, asarray=asarray, lock=lock)
 
 
+def subs(task, substitution):
```

Comment:
I'd prefer a different name, just to make it clear what `subs` function this is when glancing at the code.